tests/test_data_processor.py

```diff
@@ -349,6 +401,10 @@ def test_first_update_no_data_packets(self, data_processor):
                         estOrientQuaternionX=0.2,
                         estOrientQuaternionY=0.3,
                         estOrientQuaternionZ=0.4,
+                        estAttitudeUncertQuaternionW=0.01,
+                        estAttitudeUncertQuaternionX=0.01,
+                        estAttitudeUncertQuaternionY=0.01,
+                        estAttitudeUncertQuaternionZ=0.01,
```

Comment:
lol we really need a helper method to generate data packets like we have in test_logger, this must've been atrocious to edit